apps/notification/mutation.py

```diff
@@ -44,5 +44,37 @@ def mutate(root, info, data):
         return NotificationStatusUpdate(result=instance, ok=True, errors=None)
 
 
+class AssignmentStatusUpdate(PsGrapheneMutation):
+    class Arguments:
+        id = graphene.ID(required=True)
+        is_done = graphene.Boolean(required=True)
+    model = Assignment
+    result = graphene.Field(AssignmentType)
+    permissions = []
+
+    @classmethod
+    def perform_mutate(cls, root, info, **kwargs):
+        instance = cls.model.objects.get(id=kwargs['id'])
+        instance.is_done = kwargs['is_done']
+        instance.save(update_fields=['is_done'])
+        return cls(result=instance, ok=True)
```

Comment:
For update let's do a regular mutation
```suggestion
class AssignmentUpdate(GrapheneMutation):
    class Arguments:
        id = graphene.ID(required=True)
        data = AssignmentInputType(required=True)

    model = Assignment
    serializer_class = AssignmentSerializer
    result = graphene.Field(AssignmentType)
    permissions = []

    @classmethod
    def filter_queryset(cls, qs, info):
        return Assignment.get_for(user)
```